Jellyfin.Api/Controllers/MusicGenresController.cs

```diff
@@ -63,6 +63,8 @@ public class MusicGenresController : BaseJellyfinApiController
         /// <param name="nameStartsWithOrGreater">Optional filter by items whose name is sorted equally or greater than a given input string.</param>
         /// <param name="nameStartsWith">Optional filter by items whose name is sorted equally than a given input string.</param>
         /// <param name="nameLessThan">Optional filter by items whose name is equally or lesser than a given input string.</param>
+        /// <param name="sortBy">Optional. Specify one or more sort orders, comma delimited.</param>
+        /// <param name="sortOrder">Sort Order - Ascending,Descending.</param>
```

Comment:
Just out of curiosity, won't the `optional` comment be redundant in the API documentation with nullable parameters? Can we label these two arrays as nullable, if only for informational purposes?